ts/List/index.tsx

```diff
@@ -65,6 +65,25 @@ export class ListItemMeta extends ListItemGraphic {
   protected componentName = 'list-item__meta';
 }
 
+export interface IListItemMetaTextProps {}
+export interface IListItemMetaTextState {}
+
+export class ListItemMetaText extends MaterialComponent<
+  IListItemMetaTextProps,
+  IListItemMetaTextState
+> {
+  protected componentName = 'list-item__meta';
+  protected mdcProps = [];
```

Comment:
this needs a `padded` option

Reply:
could you elaborate? as I'm kind of new to both material, preact and preact-material I was looking for `padded` in whole project to implement the same option but couldn't find anything...

Reply:
woops sorry my bad, was looking at divider--paded